components/notification/component.yaml

```
@@ -36,40 +42,44 @@ implementation:
       from email import encoders
       port = os.getenv('PORT')
       smtp_server = os.getenv('SERVER')
-      sender_email = "$(params.sender)"
-      receiver_emails = "$(params.recipients)"
-      user = os.getenv('USER')
       password = os.getenv('PASSWORD')
       tls = os.getenv('TLS')
-      path = _parsed_args.attachment_path
+      user = os.getenv('USER')
+      sender_email = _parsed_args.sender
+      receiver_emails = _parsed_args.recipients
+      attachments = _parsed_args.attachment_path.split(',')
 
       msg = MIMEMultipart()
       msg['From'] = sender_email
       msg['To'] = receiver_emails
-      msg['Subject'] = "$(params.subject)"
-      msg.attach(MIMEText("$(params.body)"))
+      msg['Subject'] = _parsed_args.subject
+      msg.attach(MIMEText(_parsed_args.body))
       if tls == 'True':
           context = ssl.create_default_context()
           server = smtplib.SMTP_SSL(smtp_server, port, context=context)
       else:
           server = smtplib.SMTP(smtp_server, port)
       if password != '':
           server.login(user, password)
-      part = MIMEBase('application', "octet-stream")
-      with open(path, 'rb') as file:
-          part.set_payload(file.read())
-      encoders.encode_base64(part)
-      part.add_header('Content-Disposition',
-                      'attachment; filename="{}"'.format(Path(path).name))
-      msg.attach(part)
+      for path in attachments:
+          part = MIMEBase('application', "octet-stream")
+          with open(path, 'rb') as file:
+              part.set_payload(file.read())
+          encoders.encode_base64(part)
+          part.add_header('Content-Disposition',
+                          'attachment; filename="{}"'.format(Path(path).name))
+          msg.attach(part)
       for receiver in receiver_emails.split(' '):
+          logging.info("Sending Email to {}".format(receiver))
           server.sendmail(sender_email, receiver, msg.as_string())
       server.quit()
     args: [
-      --server, {inputValue: server},
       --subject, {inputValue: subject},
       --body, {inputValue: body},
       --sender, {inputValue: sender},
       --recipients, {inputValue: recipients},
       --attachment_path, {inputValue: attachment_path},
```

to reflect your changes

```suggestion
      --attachment-path, {inputValue: attachment_path},
```